README.md

```diff
@@ -164,3 +164,22 @@ To run the tests, simply run `./manage.py test <test_module> --settings=config.s
 ```
 ./manage.py test parflow_data_management.scheduler.tests.test_permissions
 ```
+
+## Develop with Docker
+
+### Initial Setup
+1. Run `docker-compose run --rm django ./manage.py migrate`
```

Comment:
This required `sudo` for me. Might be worth adding a note above the commands or including it in the command

Reply:
Thats is because you haven't setup the `docker` [group](https://docs.docker.com/engine/install/linux-postinstall/), I think we can assume this setup.